ts-redis/README.md

```diff
@@ -0,0 +1,39 @@
+# ORM Sample App
+
+## Run the app locally
+
+## Compile and Deploy with Cloudcc
+
+run the terminal commands:
+```sh
+# Compile the app
+tsc && klotho ./dist --app sample-ts-redis
+
+# npm install
+npm --prefix ./compiled install
+
+# Deploy
+pulumi up -w ./compiled -s klotho-redis-example
```

Comment:
Before this, need to set the `aws:region` right?

Reply:
right, will add the pulumi config set commands